core/string/locales.h

```diff
@@ -357,632 +357,1358 @@ static const char *country_names[][2] = {
-	{ "sli", "Lower Silesian" },
-	{ "sly", "Selayar" },
-	{ "sm", "Samoan" },
-	{ "sma", "Southern Sami" },
-	{ "smj", "Lule Sami" },
-	{ "smn", "Inari Sami" },
-	{ "sms", "Skolt Sami" },
-	{ "sn", "Shona" },
-	{ "snk", "Soninke" },
-	{ "so", "Somali" },
-	{ "sog", "Sogdien" },
-	{ "son", "Songhai" },
-	{ "sq", "Albanian" },
-	{ "sr", "Serbian" },
-	{ "srn", "Sranan Tongo" },
-	{ "srr", "Serer" },
-	{ "ss", "Swati" },
-	{ "ssy", "Saho" },
-	{ "st", "Southern Sotho" },
-	{ "stq", "Saterland Frisian" },
-	{ "su", "Sundanese" },
-	{ "suk", "Sukuma" },
-	{ "sus", "Susu" },
-	{ "sux", "Sumerian" },
-	{ "sv", "Swedish" },
-	{ "sw", "Swahili" },
-	{ "swb", "Comorian" },
-	{ "swc", "Congo Swahili" },
-	{ "syc", "Classical Syriac" },
-	{ "syr", "Syriac" },
-	{ "szl", "Silesian" },
-	{ "ta", "Tamil" },
-	{ "tcy", "Tulu" },
-	{ "te", "Telugu" },
-	{ "tem", "Timne" },
-	{ "teo", "Teso" },
-	{ "ter", "Tereno" },
-	{ "tet", "Tetum" },
-	{ "tg", "Tajik" },
-	{ "th", "Thai" },
-	{ "the", "Chitwania Tharu" },
-	{ "ti", "Tigrinya" },
-	{ "tig", "Tigre" },
-	{ "tiv", "Tiv" },
-	{ "tk", "Turkmen" },
-	{ "tkl", "Tokelau" },
-	{ "tkr", "Tsakhur" },
-	{ "tl", "Tagalog" },
-	{ "tlh", "Klingon" },
-	{ "tli", "Tlingit" },
-	{ "tly", "Talysh" },
-	{ "tmh", "Tamashek" },
-	{ "tn", "Tswana" },
-	{ "to", "Tongan" },
-	{ "tog", "Nyasa Tonga" },
-	{ "tpi", "Tok Pisin" },
-	{ "tr", "Turkish" },
-	{ "tru", "Turoyo" },
-	{ "trv", "Taroko" },
-	{ "ts", "Tsonga" },
-	{ "tsd", "Tsakonian" },
-	{ "tsi", "Tsimshian" },
-	{ "tt", "Tatar" },
-	{ "ttt", "Muslim Tat" },
-	{ "tum", "Tumbuka" },
-	{ "tvl", "Tuvalu" },
-	{ "tw", "Twi" },
-	{ "twq", "Tasawaq" },
-	{ "ty", "Tahitian" },
-	{ "tyv", "Tuvinian" },
-	{ "tzm", "Central Atlas Tamazight" },
-	{ "udm", "Udmurt" },
-	{ "ug", "Uyghur" },
-	{ "uga", "Ugaritic" },
-	{ "uk", "Ukrainian" },
-	{ "umb", "Umbundu" },
-	{ "unm", "Unami" },
-	{ "ur", "Urdu" },
-	{ "uz", "Uzbek" },
-	{ "vai", "Vai" },
-	{ "ve", "Venda" },
-	{ "vec", "Venetian" },
-	{ "vep", "Veps" },
-	{ "vi", "Vietnamese" },
-	{ "vls", "West Flemish" },
-	{ "vmf", "Main-Franconian" },
-	{ "vo", "Volapük" },
-	{ "vot", "Votic" },
-	{ "vro", "Võro" },
-	{ "vun", "Vunjo" },
-	{ "wa", "Walloon" },
-	{ "wae", "Walser" },
-	{ "wal", "Wolaytta" },
-	{ "war", "Waray" },
-	{ "was", "Washo" },
-	{ "wbp", "Warlpiri" },
-	{ "wo", "Wolof" },
-	{ "wuu", "Wu Chinese" },
-	{ "xal", "Kalmyk" },
-	{ "xh", "Xhosa" },
-	{ "xmf", "Mingrelian" },
-	{ "xog", "Soga" },
-	{ "yao", "Yao" },
-	{ "yap", "Yapese" },
-	{ "yav", "Yangben" },
-	{ "ybb", "Yemba" },
-	{ "yi", "Yiddish" },
-	{ "yo", "Yoruba" },
-	{ "yrl", "Nheengatu" },
-	{ "yue", "Yue Chinese" },
-	{ "yuw", "Papua New Guinea" },
-	{ "za", "Zhuang" },
-	{ "zap", "Zapotec" },
-	{ "zbl", "Blissymbols" },
-	{ "zea", "Zeelandic" },
-	{ "zen", "Zenaga" },
-	{ "zgh", "Standard Moroccan Tamazight" },
-	{ "zh", "Chinese" },
-	{ "zu", "Zulu" },
-	{ "zun", "Zuni" },
-	{ "zza", "Zaza" },
-	{ nullptr, nullptr }
+static const char *language_list[][3] = {
```

Comment:
Why is this still modified if you added `native_locale_list`?

Reply:
That was a leftover from the previous commit, I fixed it.